### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by manfredt27**
screenshot from 2015-10-26 15 20 06
screenshot from 2015-10-26 15 20 14

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -1,5 +1,35 @@
...
+    @user = User.find(params[:id])
+  end
+
+  def get_lost_reports
+    @lost_reports = @user.lost_reports
+  end
+
+  def user_params
+    params.require(:user).permit(
+        :name,
```
Comment (jramscr) [inline app/controllers/users_controller.rb]: Indentation.

File: app/views/main/_lost_pet.html.haml [hunk 2]
```diff
@@ -35,6 +35,7 @@
                 = image_tag("phone_icon.svg", class: "lostPet__information__detail__image")
                 .lostPet__information__detail__text
                   %p= lost_report.phone_numbers.join(" / ")
-                  %b= lost_report.owner
```
Comment (jramscr) [inline app/views/main/_lost_pet.html.haml]: This line was unnecessary?

Note: I realized that all the `p`, `b`, and `span` doesn't have one space between them and `=`, this is a haml rule or something?

Comment (manfredt27) [inline app/views/main/_lost_pet.html.haml]: I changed that line for a link to the user's profile down below `= link_to lost_report.owner, user_path(lost_report.user)'

And not sure if that's a haml rule, I think it still works with a space in the middle but I usually see it without it

File: app/views/users/_form.html.erb [hunk 3]
```diff
@@ -0,0 +1,21 @@
+<%= form_for(current_user, html: { class: 'job-manager-form'}) do |f| %>
+  <%= render 'shared/flash_messages' %>
+
+  <fieldset>
+    <label>Nombre <span class="required">*</span></label>
+    <div class="field">
+      <%= f.text_field :name, id:'nombre', class:"form-control", autofocus: true  %>
```
Comment (jramscr) [inline app/views/users/_form.html.erb]: This `id` is called `nombre` for some reason or should be named like `name`?

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+        <% if @user == current_user %>
+          <div class="row">
+            <%= link_to 'Editar', edit_user_path(id: current_user.id), class: "btn btn-sm btn-warning" %>
+          </div>
+        <% end %>
+      </div>
+    </div>
+    <br>
+    <div class="row">
+      <h1>Mascotas desaparecidad de <%= @user.name %></h1>
```
Comment (jramscr) [inline app/views/users/show.html.erb]: Fix typo here. `desaparecidad -> desaparecidas`

File: app/views/main/_lost_pet.html.haml (hunk 2 above)
Comment (jramscr) [inline app/views/main/_lost_pet.html.haml]: I haven't seem the `link_to`, awesome.

**Pull request merged**